tron/trondaemon.py

```diff
@@ -85,11 +85,23 @@ def run(self):
                 signal.SIGUSR1: self._handle_debug,
             }
             signal.pthread_sigmask(signal.SIG_BLOCK, signal_map.keys())
-
-            self._run_mcp()
+            log.info("Starting the setup process of mcp, api, manhole and runnning the reactor")
+            self._run_mcp(tron_start_time=tron_start_time)
+            log.info(
+                f"The master control plane has finished setup. Time elapsed since Tron started {time.time() - tron_start_time}s"
+            )
             self._run_www_api()
+            log.info(
+                f"The Tron API has finished setup. Time elapsed since Tron started {time.time() - tron_start_time}s"
+            )
             self._run_manhole()
+            log.info(
+                f"The manhole has finished setup. Time elapsed since Tron started {time.time() - tron_start_time}s"
+            )
             self._run_reactor()
+            log.info(
+                f"Twisted reactor has started. The Tron API should be up and ready now to receive requests. Time elapsed since Tron started {time.time() - tron_start_time}s"
+            )
```

Comment:
```suggestion
            log.info(
                f"Twisted reactor has started. The Tron API should be up and ready now to receive requests. Time elapsed since Tron started: {time.time() - tron_start_time}s"
            )
            log.info("Setup complete!")
```